storage/filesystem/internal/dotgit/dotgit_setref_norwfs.go

```diff
@@ -0,0 +1,17 @@
+// +build norwfs
+
+package dotgit
+
+import "gopkg.in/src-d/go-git.v4/plumbing"
+
+func (d *DotGit) setRef(fileName, content string, old *plumbing.Reference) error {
```

Comment:
Can you add some comments here explaining the reason of this method (like on the PR description)?